modules/sql-engine/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     implementation libs.classgraph
     implementation libs.javassist
     implementation libs.auto.service.annotations
+    implementation libs.jsr305
```

Comment:
Why do we need JSR305?

Reply:
for autogenerated code (@Nullable annotations) there is no way to change it into jetbrains implementation, previously this dependency was linked transiently